website/pages/en/seminar.js

```diff
@@ -129,19 +129,11 @@ function Seminar(props) {
 						<div className='col-md-6 second'>
 							<h4 className='mb-2'>How should I prepare for the call?</h4>
 							<p>
-								For the February 25th session, it wil be helpful to know a bit about Polkadot's use of
-								Phragmen in Proof of Stake. You may enjoy this{' '}
-								<a href="https://research.web3.foundation/en/latest/polkadot/NPoS/">
-									informal overview
-								</a>{' '}
-								or this more{' '}
-								<a href="https://research.web3.foundation/en/latest/polkadot/NPoS/4.%20Sequential%20Phragm%C3%A9n%E2%80%99s%20method.html">
-									mathematical writeup
-								</a>.
-								If you'd like to join a testnet, please build the code on Substrate's{' '}
-								<a href="https://github.com/paritytech/substrate/tree/kiz-offchain-phragmen-4">
-									kiz-offchain-phragmen-4 branch
-								</a>.
+								For the March 3rd call, there are no specific preparation steps. You may find it
```

Comment:
2nd or 3rd?

Reply:
3rd is correct. They are every Tuesday. The PR is named after the date I made it because I make one of these about once a week.